models/migrations/migrations.go

```diff
@@ -82,6 +82,8 @@ var migrations = []Migration{
 	NewMigration("create user column allow create organization", createAllowCreateOrganizationColumn),
 	// V16 -> v17
 	NewMigration("create repo unit table and add units for all repos", addUnitsToTables),
+	// v17 -> v18
+	NewMigration("set protect branches updated with created", setProtectedBranchUpdatedWithCreated),
```

Comment:
No. Sync on a migration is better safe. I have found the original method's bug, see #871. When you upgrade from an old version which haven't a column external_tracker_url, then a new version add this column by Sync not migration. but another new version will use this column, then the bug occupied.